amivapi/blacklist/emails.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# license: AGPLv3, see LICENSE for details. In addition we strongly encourage
+#          you to buy us beer if we meet and you like the software.
+"""Logic to send emails on blacklist changes (deleted items don't triggger an
```

Comment:
Docstrings should consist of an Title line, separated by spaces from the body, e.g.:

```
"""This is a proper docstring.

This is some more description., Not that the ending quotes are on the next line,
and every sentence ends with a full stop.
"""
```